python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -247,6 +281,62 @@ def __init__(self, **kwargs):
         for flag in self.config.badSourceFlags:
             if flag not in self.schema:
                 raise pipeBase.InvalidQuantumError("Field %s not in schema" % flag)
+
+        if self.config.doWriteMetrics:
```

Comment:
As I said on the analysis_tools PR, I'm worried about calling these "metrics" as well when they are stored and handled quite differently from both `lsst.verify.Metrics` and `analysis_tools` metrics.

Reply:
Changed to `spatiallySampledMetrics`